src/algos/keyboards.ts

```diff
@@ -39,7 +39,7 @@ export class manager extends AlgoManager {
   public matchTerms: string[] = [
     '⌨️',
     'ajazz',
-    'akko',
+    // 'akko',
```

Comment:
This has a _lot_ of false positives, particularly with some Little Witch Academia character? Idk, it pulls in a lot of anime fanart.